pkg/logs/internal/util/test_utils.go

```diff
@@ -5,25 +5,9 @@
 
 package util
 
-import "github.com/DataDog/datadog-agent/pkg/logs/sources"
-
-// ConsumeSources ensures that another component is consuming the channel to prevent
-// the producer to get stuck.
-func consumeSources(sources *sources.LogSources) {
-	go func() {
-		sources := sources.GetAddedForType("foo")
-		//nolint:revive // TODO(AML) Fix revive linter
-		for range sources {
-		}
-	}()
-}
+import (
+	"github.com/DataDog/datadog-agent/pkg/logs/internal/util/testutils"
+)
 
 // CreateSources creates sources
-func CreateSources(sourcesArray []*sources.LogSource) *sources.LogSources {
-	logSources := sources.NewLogSources()
-	for _, source := range sourcesArray {
-		logSources.AddSource(source)
-	}
-	consumeSources(logSources)
-	return logSources
-}
+var CreateSources = testutils.CreateSources
```

Comment:
Why does this still need to be defined to point at `testutils`? Is there a reason you can't import `testutils` in the other locations this is needed?